src/PrestaShopBundle/Form/Admin/Sell/Product/Specification/ReferencesType.php

```diff
@@ -47,7 +47,7 @@ public function buildForm(FormBuilderInterface $builder, array $options)
             ->add('reference', TextType::class, [
                 'required' => false,
                 'label' => $this->trans('Reference', 'Admin.Global'),
-                'label_help_box' => $this->trans('Your reference code for this product. Allowed special characters: .-_#.', 'Admin.Catalog.Help'),
+                'label_help_box' => $this->trans('Your reference code for this product.', 'Admin.Catalog.Help'),
```

Comment:
I don't find this help text useful and I think it could be removed completely. The label speaks for itself.

Reply:
> I don't find this help text useful and I think it could be removed completely. The label speaks for itself.

You can use this new string that was introduced in #27216 

'Allowed special characters: %allowed_characters%' in Admin.Global